src/Repository/ForumRepository.php

```diff
@@ -165,7 +176,7 @@ public function findForumNames($names) {
     public function findForumsInCategory(ForumCategory $category) {
         /* @noinspection SqlDialectInspection */
         $dql = 'SELECT f.id, f.name FROM '.Forum::class.' f '.
-            'WHERE f.category = :category '.
+            'WHERE f.category = :category and f.id > 0'.
```

Comment:
comment here or somewhere else why f.id > is necessary would be nice

Reply:
Fixed in update